agent/cache-types/streaming.go

```diff
@@ -89,6 +91,21 @@ func (s *Subscriber) run() {
 			return
 		}
 
+		// Check to see if this is a special "reload stream" message before processing
+		// the event.
+		if event.GetReloadStream() {
+			fmt.Printf("[WARN] got ACL reset event\n")
```

Comment:
Debug? Probably shouldn't log this at WARN and if we do we should include more context and use a real logger!